src/app/(main)/page.tsx

```diff
@@ -34,7 +35,7 @@ const Home = async () => {
   );
 
   if (!landingId) {
-    return <NewTemplatePlaceholder />;
+    return <MissingData description={"Landing page id"} />
```

Comment:
we can do this in a separate PR and not something pressing!

we know that if landingId doesn't exist it's most likely because sanity studio is empty. In terms of UX we should add a nicer Welcome screen with some description on how to get started. I can create something in figma for us to use as a template and create an issue based off of that.